### ISSUE TITLE
doctrine:database:drop fails to remove sqlite file when on windows

### ISSUE THREAD

**Issue by olberger** [symfony/symfony#51309]
### Symfony version(s) affected

6.3.3

### Description

In a freshly created Symfony 6.3 project, on windows (`symfony new tata --version="6.3.*" --webapp`), in with DATABASE_URL is set in .env to use SQLite (`DATABASE_URL="sqlite:///%kernel.project_dir%/var/data.db"`), `symfony console doctrine:database:drop` fails to remove the file, whereas `symfony console doctrine:database:create` works.

One can use rm, of course, but it would be great to have both symetrical commands work.

I suspect some path issue as on windows, / and \ may not be treated the same as on Linux.

Hope this helps.

### How to reproduce

Log of a session on git-bash:
```
berger_o@PAT15278 MINGW64 ~/CSC4101/tp-01/toto (master)
$ ls var/
cache/  data.db  log/

berger_o@PAT15278 MINGW64 ~/CSC4101/tp-01/toto (master)
$ symfony console doctrine:database:drop --force
Could not drop database C:\Users\berger_o\CSC4101\tp-01\toto/var/data.db for connection named default
Warning: unlink(C:\Users\berger_o\CSC4101\tp-01\toto/var/data.db): Resource temporarily unavailable

berger_o@PAT15278 MINGW64 ~/CSC4101/tp-01/toto (master)
(reverse-i-search)`rm': ^C todo.sqlite

berger_o@PAT15278 MINGW64 ~/CSC4101/tp-01/toto (master)
$ ls var/
cache/  data.db  log/

berger_o@PAT15278 MINGW64 ~/CSC4101/tp-01/toto (master)
$ symfony console doctrine:database:drop --force
Could not drop database C:\Users\berger_o\CSC4101\tp-01\toto/var/data.db for connection named default
Warning: unlink(C:\Users\berger_o\CSC4101\tp-01\toto/var/data.db): Resource temporarily unavailable

berger_o@PAT15278 MINGW64 ~/CSC4101/tp-01/toto (master)
$ rm var/data.db

berger_o@PAT15278 MINGW64 ~/CSC4101/tp-01/toto (master)
$ ls var/
cache/  log/

berger_o@PAT15278 MINGW64 ~/CSC4101/tp-01/toto (master)
$ symfony console doctrine:database:create
Created database C:\Users\berger_o\CSC4101\tp-01\toto/var/data.db for connection named default

berger_o@PAT15278 MINGW64 ~/CSC4101/tp-01/toto (master)
$ ls var/
cache/  data.db  log/
```

And a similar one with powershell :
```
PS C:\Users\berger_o\CSC4101\tp-01\tata> ls var


    Répertoire : C:\Users\berger_o\CSC4101\tp-01\tata\var


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----        08/08/2023     11:27                cache
d-----        08/08/2023     11:28                log
-a----        08/08/2023     11:29              0 data.db


PS C:\Users\berger_o\CSC4101\tp-01\tata> symfony console doctrine:database:drop --force
Could not drop database C:\Users\berger_o\CSC4101\tp-01\tata/var/data.db for connection named default
Warning: unlink(C:\Users\berger_o\CSC4101\tp-01\tata/var/data.db): Resource temporarily unavailable
PS C:\Users\berger_o\CSC4101\tp-01\tata> rm .\var\data.db
PS C:\Users\berger_o\CSC4101\tp-01\tata> ls var


    Répertoire : C:\Users\berger_o\CSC4101\tp-01\tata\var


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----        08/08/2023     11:27                cache
d-----        08/08/2023     11:28                log


PS C:\Users\berger_o\CSC4101\tp-01\tata> symfony console doctrine:database:create
Created database C:\Users\berger_o\CSC4101\tp-01\tata/var/data.db for connection named default
PS C:\Users\berger_o\CSC4101\tp-01\tata> ls var


    Répertoire : C:\Users\berger_o\CSC4101\tp-01\tata\var


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----        08/08/2023     11:27                cache
d-----        08/08/2023     11:28                log
-a----        08/08/2023     11:30              0 data.db
```

This leads me to suspect that the issue is in the command and not the tools used.

### Possible Solution

Using rm is of course a direct way to remote the DB

**Comment by stloyd** [comment]
This has nothing to do with Symfony, it's a bug in Doctrine Bundle: https://github.com/doctrine/DoctrineBundle/issues/1658

**Comment by olberger** [comment]
Hopefully this will be resolved in DoctrineBundle 2.10.2 soon (https://github.com/doctrine/DoctrineBundle/compare/2.10.1...2.10.x), which may be there for new 6.3 projects ? 

EDIT: I was a bit puzzled by the dependency on 2.10 apparently in new projects for 6.3 (`"doctrine/doctrine-bundle" : "^2.10"`), when looking at branches of the Doctrine ORM (2.16 being stable in https://www.doctrine-project.org/projects/orm.html), but these are 2 different versioning sets

**Comment by xabbuh** [comment]
closing here then

**Comment by derrabus** [comment]
I've released version 2.10.2.

**Comment by olberger** [comment]
> I've released version 2.10.2.

Which fixes the issue as expected. Many thanks.
